airflow/task/task_runner/base_task_runner.py

```diff
@@ -85,6 +86,12 @@ def __init__(self, local_task_job):
             cfg_path = tmp_configuration_copy(chmod=0o600)
 
         self._error_file = NamedTemporaryFile(delete=True)
+        if self.run_as_user:
+            try:
+                os.chown(self._error_file.name, getpwnam(self.run_as_user).pw_uid, -1)
+            except Exception:
+                pass
```

Comment:
This should either bubble the exception or log that the chown failed. Personally I’d prefer *not* catching the exception for now and fix UX issues when someone complains.